src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -123,6 +188,14 @@ public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
      * или сообщение "dateTime can't be formatted!"
      */
     public String formatDate(LocalDateTime dateTime) {
-        return "Date can't be formatted!";
+        String result = null;
```

Comment:
do we need this variable? let's write 
return dateTime.format(...

Reply:
I`ve changed it